templates/config/config_pem.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'config/config_layout.html' %}
+
+{% block article %}
+<div class="form-group row">
+  <!-- <label for="Name" class="col-sm-2 col-form-label">Name</label> -->
+  <form class="form-inline" action="{{ url_for('pem_create') }}" method="POST">
+    <div class="col-sm-12">
+        <input type="text" id="Name" name="Name" class="form-control" placeholder="Insert Key Pair Name"/>
```

Comment:
name="Name" is not appropriate. `id` must be only one within the whole html file. `name` should be turn into more easy-to-read style like `RSAPrivKey`.